protocols/bgp/server/fsm_address_family.go

```diff
@@ -44,6 +46,7 @@ func newFSMAddressFamily(afi uint16, safi uint8, family *peerAddressFamily, fsm
 		afi:               afi,
 		safi:              safi,
 		fsm:               fsm,
+		vrf:               fsm.peer.vrf,
```

Comment:
Why don't we access the vrf using fsm.peer.vrf but make a copy of the pointer here?